hip_data_tools/common.py

```diff
@@ -18,6 +18,7 @@
                          "most_specific_location"]
 
 LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.DEBUG)
```

Comment:
can we hook this to a flag or an env?  just to avoid all consumers to start flooding cloudwatch..

Reply:
toggling via env varirable